websites/S/ShareX/presence.ts

```diff
@@ -0,0 +1,141 @@
+const presence = new Presence({
+  clientId: "888726220571811914" 
+}),
+ browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const showTimestamp: boolean = await presence.getSetting("timestamp"),
+  showButtons: boolean = await presence.getSetting("buttons"),
+presenceData: PresenceData = {
+  largeImageKey:
+    "sharex-logo",
+  smallImageKey:
+    "sharex-white-logo",
+  smallImageText: "Navigating on getsharex.com",
+  buttons: [
+  {
+    label: "View Page",
+    url: document.location.href
+  }
+]
+};
+// Main Pages
+if (document.location.pathname === "/") {
+presenceData.state = "🏠 | Browsing Home Page";
```

Comment:
You're not **required** to do this but I think you should use smallImageKey instead of emojis as no other presences does this

Reply:
Implementing this, thanks for suggesting

Reply:
I finished implementing it. Waiting for art assets to deploy. I'll edit this message uploading proof of the final product when it's done